Gemfile

```diff
@@ -2,14 +2,41 @@ source "https://rubygems.org"
 
 gemspec
 
-gem 'rspec-its'
+# Temporary hack to troubleshoot what elrayle is seeing
+gem 'rails', '~> 5.0.2'
```

Comment:
Log from engine_cart:generate from failure to end of log (rails 5.1.0):  https://gist.github.com/elrayle/2cb05cba2963dd84635f2ccf956f79c9